conf/app.conf

```
@@ -0,0 +1,4 @@
+appname = go-scraper
+runmode="${APP_RUN_MODE||dev}"
```

@hoangmirs Should we add some spaces like the other line?

```suggestion
runmode = "${APP_RUN_MODE||dev}"
```